### PR TITLE
Add Verible action with automatic review

### PR DESCRIPTION
**Pull request by wsipak**
Two successful tests here:
https://github.com/antmicro/gha-playground/pull/17
https://github.com/antmicro/gha-playground/pull/16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    extra_args: "--check_syntax=true"
+    github_token: ${{ secrets.GITHUB_TOKEN }}
+```
+
+Automatic review on PRs from external repositories
+--------------------------------------------------
+
+In GitHub Actions, workflows triggered by external repositories
+may only have read access to the main repository
+\[[docs](https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token)\].
```
Comment (tgorochowik) [inline README.md]: can we make "read access to the main repository" the link? So we don't have to have the link in in brackets.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+In order to have automatic review on external PRs, you need to create 2 workflows.
```
Comment (tgorochowik) [inline README.md]: * review -> reviews
* 2 -> two

File: README.md (hunk 1 above)
Comment (wsipak) [reply README.md]: Yes, done.

File: README.md (hunk 2 above)
Comment (wsipak) [reply README.md]: Done

**Review by tgorochowik — APPROVED** [review]
LGTM!

**Pull request merged**